File: library/src/main/java/name/zeno/android/util/ZHtml.java
package name.zeno.android.util;

import android.text.Html;

/**
 * @author 陈治谋 (513500085@qq.com)
 * @since 2017/9/4
 */
public abstract class ZHtml
{
  public static CharSequence fromHtml(String source)
  {
    return Html.fromHtml(source);
  }
}

File: zako/src/main/java/cn/izeno/android/util/ZHtml.kt
package cn.izeno.android.util

import android.text.Html

/**
 * @author 陈治谋 (513500085@qq.com)
 * @since 2017/9/4
 */
object ZHtml {
  fun fromHtml(source: String): CharSequence {
    return Html.fromHtml(source)
  }
}
